KsApi/fragments/BackingFragment.graphql

```diff
@@ -0,0 +1,32 @@
+fragment BackingFragment on Backing {
+  amount {
+    ...MoneyFragment
+  }
+  backer {
+    ...UserFragment
+  }
+  backerCompleted
+  bonusAmount {
+    ...MoneyFragment
+  }
+  cancelable
+  creditCard: paymentSource {
```

Comment:
is `paymentSource` something baked into the `GraphAPI` when the script runs?

Reply:
The property is called `paymentSource` in the schema, we do this to alias it as `creditCard`.